grakn-test/src/test/java/ai/grakn/generator/ConceptIds.java

```diff
@@ -22,7 +22,7 @@
 import ai.grakn.concept.ConceptId;
 
 /**
- * Generator that generates totally concept IDs
+ * Generator that generates totally random concept IDs
  */
 public class ConceptIds extends AbstractGenerator<ConceptId> {
```

Comment:
I don't think we can 😞 
The`grakn-graph` property tests must be in a module that depends on `grakn-test-tools` (because that's where the generators are).